### PR TITLE
Initial set of linting rules

### PR DESCRIPTION
**Pull request by norisalsaadie**

### CODE REVIEW

**Review by will-byrne — APPROVED** [review]

**Comment by thewilkybarkid** [comment]
Are you doing this just for Reviewer? If so, this repo needs to be named as such.

**Comment by erezmus** [comment]
intention is for it to be libero wide

**Review by NuclearRedeye — APPROVED** [review]
Looks good as an initial stake in the ground. Think looking forward that a README including some details on how to setup and use the library in another project would be good!

**Comment by erezmus** [comment]
> Looks good as an initial stake in the ground. Think looking forward that a README including some details on how to setup and use the library in another project would be good!

oh yeah, forgot that bit. will add now

**Review by NuclearRedeye — APPROVED** [review]

**Comment by norisalsaadie** [comment]
Can we add a github action to publish this to npm when tagged on master?

**Comment by giorgiosironi** [comment]
> Can we add a github action to publish this to npm when tagged on master?

In principle yes, but in a separate PR after this is reviewed and eventually approved.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # eslint-config
 A set of standard eslint rules to use on Typescript repositories
+
+## Getting Started
+
+Install the package:
+```sh
+npm install --save-dev @libero/eslint-config
```
Comment (giorgiosironi) [inline README.md:8]: specifying a version here? Just the latest stable?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+```
+
+Or using yarn:
+
+```sh
+yarn add --dev @libero/eslint-config
+```
+
+In your package root, create an `.eslintrc.js` file with following contents:
```
Comment (giorgiosironi) [inline README.md:17]: how difficult it is to add a `node_modules/.bin/something` that does this?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@libero/eslint-config",
+  "version": "0.0.1",
```
Comment (giorgiosironi) [inline package.json:3]: isn't the first version ever released usually `0.1.0`?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "build": "tsc",
+    "prettier": "prettier --write 'src/**/*ts'"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.17.0",
+    "@typescript-eslint/parser": "^2.17.0",
+    "eslint": "^6.8.0",
```
Comment (giorgiosironi) [inline package.json:19]: Since this package brings in `eslint`, it's not just `config`

Comment (giorgiosironi) [reply package.json:19]: Or does it? It's also in the `peerDependencies `which suggests the project using this will require `eslint` on his own

**Review by giorgiosironi — COMMENTED** [review]
There are no tests that this does what you want it to do - the scope of those can be negotiated but I would expect something like `npm test` to execute `eslint` with this configuration over a set of sample files

**Pull request merged**